resources/assets/less/bem/mp-history-event.less

```diff
@@ -0,0 +1,98 @@
+/**
+ *    Copyright 2016 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+.mp-history-event {
+  display: flex;
+  align-items: baseline;
+
+  max-width: 640px;
+  margin-left: auto;
+  margin-right: auto;
+
+  background-color: @grey-e;
+
+  padding-bottom: @spacing * 0.5;
+
+  .mp-history-content__show-more-box + &, .mp-history-game + & {
```

Comment:
I was thinking something something like this: http://codepen.io/nanaya/pen/ZOrBzG